analytics/google/experience-accelerator/src/GAClient.ts

```diff
@@ -25,13 +22,10 @@ export class GAClient extends Client {
         let augmentedCidEid = this.getAugmentedCidEid(event);
 
         this.emit('create', this.trackingId, 'auto', namespace ? {namespace} : null);
-        this.emit(prefix + 'set', 'dimension' + this.candidateIdDimension, augmentedCidEid);
-        this.emit(prefix + 'set', 'dimension' + this.userIdDimension, augmentedUid);
-        this.emit(prefix + 'set', 'dimension' + this.sessionIdDimension, augmentedSid);
 
         this.emit(prefix + 'send', 'event', { 
             'eventCategory': 'evolvids',
-            'eventAction': augmentedCidEid,
+            'eventAction': 'evolv-event' + (augmentedCidEid ? ':' + augmentedCidEid : ''),
```

Comment:
Maybe worth a constant for `evolv-event`.

Reply:
Will do -- got another PR coming in that I'll add it to